reactComponents/src/searchbar/tokenSearch/SearchFilters.tsx

```diff
@@ -28,7 +40,11 @@ export const SearchFilters = () => {
         <AccordionItemHeading>
           <AccordionItemButton >
             <div tw="p-4 flex">
-              <div tw="font-bold">Filter Networks:</div>  &nbsp; Searching {networkCount} networks and {exchangeCount} exchanges
+              <div tw="font-bold">Filter Networks:</div>&nbsp;Searching {
```

Comment:
https://github.com/Rome-Blockchain-Labs/terminal-library/pull/12 doesnt use tailwind anymore

Reply:
Which branch do you review now?
I only work on search-bar branch.